### PR TITLE
Add Discord Chat Link

### PR DESCRIPTION
**Pull request by mauerbac**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # AWS SDK for iOS
-
+[![DiscordChat](https://img.shields.io/discord/308323056592486420?logo=discord")](https://discord.gg/jWVbPfC)
```
Comment (jpignata) [inline README.md:2]: This link doesn't render properly - same issue - stray quote mark.

Comment (jpignata) [reply README.md:2]: Seems like we can punch this up a little bit? Why would I go to Discord?

**Comment by jpignata** [comment]
Please open a similar pull for https://github.com/aws-amplify/amplify-ios.

**Comment by palpatim** [comment]
Actually, this was already merged in on #2586. Closing this one.

**Pull request closed without merging**